xfrd-tcp.c

```diff
@@ -989,9 +1309,19 @@ xfrd_tcp_pipe_release(struct xfrd_tcp_set* set, struct xfrd_tcp_pipeline* tp,
 		event_del(&tp->handler);
 	tp->handler_added = 0;
 
+#ifdef HACVE_TLS_1_3
```

Comment:
Spelling error in ifdef HAVE_TLS_1_3.

```suggestion
#ifdef HAVE_TLS_1_3
```